docs/tutorials/permissions.rst

```diff
@@ -1,136 +1,28 @@
-
-In this case the user id is: ``basicauth:631c2d625ee5726172cf67c6750de10a3e1a04bcd603bc9ad6d6b196fa8257a6``
+Let's use one of the :ref:`application examples <app-examples-blog>`: using *Kinto* as
+a storage API for a blog application.
 
 .. note::
 
-    In case of sharing, users need a way to share their user id with
-    people that needs to give them permission.
+    Some general details are provided in the :ref:`concepts` and :ref:`api/permissions`
+    sections. Make sure you are familiar with main concepts before starting this.
 
 
-Object permissions
-==================
-
-.. note::
-
-    Please make sure to have read the :ref:`permission documentation <permissions>`
-    first.
-
-For instance, we can look at our *personal bucket* permissions.
-
-.. code-block:: http
-
-    $ http GET http://localhost:8888/v1/buckets/default --auth user:pass
-    HTTP/1.1 200 OK
-    Access-Control-Expose-Headers: Backoff, Retry-After, Alert, Content-Length, Last-Modified, ETag
-    Content-Length: 187
-    Content-Type: application/json; charset=UTF-8
-    Date: Thu, 16 Jul 2015 09:59:30 GMT
-    Etag: "1437040770742"
-    Last-Modified: Thu, 16 Jul 2015 09:59:30 GMT
-    Server: waitress
-
-    {
-        "data": {
-            "id": "46524be8-0ad7-3ac6-e260-71f8993feffa",
-            "last_modified": 1437040770742
-        },
-        "permissions": {
-            "write": [
-                "basicauth:631c2d625ee5726172cf67c6750de10a3e1a04bcd603bc9ad6d6b196fa8257a6"
-            ]
-        }
-    }
-
-
-Similarly, the permissions of a collection can be obtained with a ``GET``:
-
-.. code-block:: http
-
-    $ http GET http://localhost:8888/v1/buckets/default/collections/tasks --auth user:pass
-    HTTP/1.1 200 OK
-    Access-Control-Expose-Headers: Backoff, Retry-After, Alert, Content-Length, Last-Modified, ETag
-    Content-Length: 156
-    Content-Type: application/json; charset=UTF-8
-    Date: Thu, 16 Jul 2015 10:00:30 GMT
-    Etag: "1437040830468"
-    Last-Modified: Thu, 16 Jul 2015 10:00:30 GMT
-    Server: waitress
-
-    {
-        "data": {
-            "id": "tasks",
-            "last_modified": 1437040830468
-        },
-        "permissions": {
-            "write": [
-                "basicauth:631c2d625ee5726172cf67c6750de10a3e1a04bcd603bc9ad6d6b196fa8257a6"
-            ]
-        }
-    }
-
-
-Managing object permissions
-===========================
-
-Permissions can be specified during the creation of an object, and can
-later be updated using PUT or PATCH.
-
-.. note::
+Basic permission setup
+======================
 
-   The user that updates the permissions is always given the ``write``
-   permission, in order to prevent loosing ownership on the object.
+A ``servicedenuages-blog`` bucket will contain two collections: ``articles`` and
+``comments``.
 
-A :ref:`blog bucket <permissions_usecases>` could be created with the following to
-give read access to everyone.
+Let's start with the bucket with read access to everyone.
```

Comment:
> Let's start by giving everybody read access to the bucket.

That said, according to `docs/api/permissions.rst`, the `system.Authenticated` flag means all *authenticated* users (not anonymous). Assuming we don't want to change the example, we should be clear about what it means:
> Let's start by giving all authenticated users read access to the bucket.